test/testsuites/cscpackagetests.go

```diff
@@ -54,31 +53,24 @@ func TestCscWithNonExistingPackage(t *testing.T) {
 // nonexistent namespace.
 func configuringStateWhenPackageNameDoesNotExist(t *testing.T) {
 	namespace, err := test.NewTestCtx(t).GetNamespace()
-	if err != nil {
-		t.Fatalf("Could not get namespace: %v", err)
-	}
+	require.NoError(t, err, "Could not get namespace")
 
 	// Check that the catalogsourceconfig with an non-existing package eventually reaches the
 	// configuring phase with the expected message
 	expectedPhase := "Configuring"
 	expectedMessage := fmt.Sprintf("Still resolving package(s) - %v. Please make sure these are valid packages.", nonExistingPackageName)
 	err = helpers.WaitForExpectedPhaseAndMessage(test.Global.Client, cscName, namespace, expectedPhase, expectedMessage)
-	if err != nil {
-		t.Fatalf("CatalogSourceConfig never reached expected phase/message, expected %v/%v", expectedPhase, expectedMessage)
-	}
+	assert.NoError(t, err, fmt.Sprintf("CatalogSourceConfig never reached expected phase/message, expected %v/%v", expectedPhase, expectedMessage))
 }
 
 // childResourcesNotCreated checks that once a CatalogSourceConfig with a wrong package name
 // is created that all expected runtime objects are not created.
 func childResourcesNotCreated(t *testing.T) {
 	// Get test namespace.
 	namespace, err := test.NewTestCtx(t).GetNamespace()
-	if err != nil {
-		t.Fatalf("Could not get namespace: %v", err)
-	}
+	require.NoError(t, err, "Could not get namespace")
+
 	// Check that the CatalogSourceConfig's child resources were not created.
 	err = helpers.CheckCscChildResourcesDeleted(test.Global.Client, cscName, namespace, namespace)
-	if err != nil {
-		t.Fatal(err)
-	}
+	assert.NoError(t, err, "Child Resources of CatalogSourceConfig were created")
```

Comment:
Is it possible to frame this error message in a way that sounds like an error more? I'm thinking if this message shows up during CI testing it might not sound like an error and can get confusing. 
How about `Child resources of CatalogSourceConfig were unexpectedly created`? 
Also `nit`: `resources` with `r`.

Reply:
+1 for @anik120's suggested error message